src/routes/login.ts

```diff
@@ -78,8 +79,20 @@ export const createLoginRoute: RouteCreator =
             flow: verificationFlow.id,
             message: JSON.stringify(loginFlow.ui.messages),
           })
+
+          const baseUrl = req.path.split("/")
+          // get rid of the last part of the path (e.g. "login")
+          baseUrl.pop()
```

Comment:
This is maybe a bit overkill and just a simple res.redirect("verification") might've been suffice. But I added this in case the application was hosted with a base URL added to the router.